src/test/helper.ts

```diff
@@ -49,7 +49,8 @@ export async function updateSettings(
   value: unknown
 ): Promise<void> {
   const ansibleConfiguration = vscode.workspace.getConfiguration('ansible');
-  return ansibleConfiguration.update(setting, value, false);
+  const useGlobalSettings = true;
```

Comment:
I think that the naming could be improved to provide some context into what is the effect of using this value.
https://github.com/ansible/vscode-ansible/pull/382#discussion_r786823160

Reply:
Also, with this change, wouldn't it modify the semantics of the whole function? This could be reflected in the function name — what do the callers of `updateSettings()` expect to happen? By changing the behavior here you change the semantical meaning of the function but the callers still expect to update the same settings (which are not the same anymore).

Is it `updateUserGlobalSettings()` now? `updateProjectSpecificSettings()`?

P.S. It wouldn't hurt to have a jsdoc comment here.

Reply:
I am not in favor of changing the name of the function. It does what it says. I am running out of names for the variable as well, but still, would try to come up with a better name. 

Also, thanks for the suggestion of the `jsdoc` comment. Will do that in a separate PR.